handel/src/services/apigateway/proxy/proxy-passthrough-deploy-type.ts

```diff
@@ -77,6 +79,22 @@ async function getCompiledApiGatewayTemplate(stackName: string, ownServiceContex
         handlebarsParams.customDomains = await apigatewayCommon.getCustomDomainHandlebarsParams(ownServiceContext, serviceParams.custom_domains);
     }
 
+    const warmup: WarmupConfig = getParam(serviceParams, 'warmup', 'warmup', undefined);
```

Comment:
Just as an FYI, I'm hopefully going to be remove the old deprecated configuration syntax for proxy APIs here in the next few days (almost everyone has moved to the new configuration style), so we'll be able to get rid of that `getParam` method to make things a tiny bit clearer.